OracleGoldenGate/README.md

```diff
@@ -1,8 +1,13 @@
-Oracle GoldenGate on Docker
-===============
+# Oracle GoldenGate on Docker
+
 Sample Docker build files to provide an installation of Oracle GoldenGate for DevOps users.
 
 ## Oracle GoldenGate 19c and earlier
 
 Provides Docker build files to create a Docker image for Oracle GoldenGate versions 12c through 19c.
```

Comment:
Please replace Docker with container for the second instance, i.e. "Provides Docker build files to create a container image..."